tests/01_03_52-disable_lock_whitelist.js

```diff
@@ -1,121 +1,107 @@
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    const addr = await rskClient.eth.personal.importRawKey(WHITELIST_CHANGE_PK, '');
-    expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
-    await rskClient.eth.personal.unlockAccount(addr, '');
-    // can disable the whitelist
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(200),
-      (disableResult) => expect(Number(disableResult)).to.equal(1),
-      WHITELIST_CHANGE_ADDR)();
-
-    // disable whitelist doesn't work the second time
-    await utils.sendTxWithCheck(
-      rskClient.rsk.bridge.methods.setLockWhitelistDisableBlockDelay(10),
-      (disableResult) => expect(Number(disableResult)).to.equal(-1),
-      WHITELIST_CHANGE_ADDR)();
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // address is still not able to send btc to bridge after 100 blocks
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }], { fails: true });
-    await utils.waitForBtcToReturn(addresses.btc);
-
-    await btcClient.generate(100);
-    await wait(500);
-    await rskClient.fed.updateBridge();
-    await rskUtils.mineAndSync(rskTxHelpers);
-    await wait(500);
-
-    // after 200 blocks the whitelist period has ended and we can send money to the bridge
-    await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }]);
+    await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, MINIMUM_PEGIN_VALUE_IN_BTC);
+    const btcAddressBalanceAfterFirstPegin = Number(btcToSatoshis(await btcTxHelper.getAddressBalance(btcAddressInfo.address)));
+    expect(btcAddressBalanceAfterFirstPegin).to.be.equal(btcAddressBalanceInitial - MINIMUM_PEGIN_VALUE_IN_SATS - FEE_IN_SATOSHI);
+    const federationAddressBalanceAfterFirstPegin = Number(await btcTxHelper.getAddressBalance(federationAddress));
+    expect(federationAddressBalanceAfterFirstPegin).to.be.equal(federationAddressBalanceInitial + MINIMUM_PEGIN_VALUE_IN_BTC)
+    
+    // wait for the btc to come back so we can use the sendPegin method again
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+    const btcAddressBalanceAfterFirstTriggerRelease = Number(btcToSatoshis(await btcTxHelper.getAddressBalance(btcAddressInfo.address)));
+    expect(btcAddressBalanceInitial - btcAddressBalanceAfterFirstTriggerRelease).to.be.at.most(FEE_IN_SATOSHI * 2)
+    const federationAddressBalanceAfterFirstTriggerRelease = Number(await btcTxHelper.getAddressBalance(federationAddress));
+    expect(federationAddressBalanceAfterFirstTriggerRelease).to.be.equal(federationAddressBalanceInitial)
+
+    // disable whitelisting after 20 blocks
+    const unlocked = await rskUtils.getUnlockedAddress(rskTxHelper, WHITELIST_CHANGE_PK, WHITELIST_CHANGE_ADDR);
```

Comment:
What is this used for?

Reply:
Imports the private key and unlocks the rskAddress, asserting that the rskAddress returned by the private key import is the same as the rskAddress provided.

Reply:
oh I see, thanks